packages/components/src/radio-group/stories/index.story.tsx

```diff
@@ -1,32 +1,41 @@
 /**
- * WordPress dependencies
+ * External dependencies
  */
-import { useState } from '@wordpress/element';
+import type { Meta, StoryFn } from '@storybook/react';
 
 /**
  * Internal dependencies
  */
-import { Radio, RadioGroup } from '../../radio-group-ariakit/';
+import { RadioGroup } from '..';
+import { Radio } from '../radio';
+import type { RadioProps } from '../types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
 
-export default {
-	title: 'Components (Deprecated)/RadioGroup',
+const meta: Meta< typeof RadioGroup > = {
 	component: RadioGroup,
-	subcomponents: { Radio },
```

Comment:
We should keep the subcomponents prop around (even if TS complains about it), similarly to what we've done in other components ([example](https://github.com/WordPress/gutenberg/blob/d5ade48d001d56d6fe43331aef4d98d25b5ce65c/packages/components/src/dropdown/stories/index.story.tsx#L16-L17))